packages/react-core/src/components/Page/PageToggleButton.tsx

```diff
@@ -10,12 +10,15 @@ export interface PageToggleButtonProps extends ButtonProps {
   isNavOpen?: boolean;
   /** Callback function to handle the side nav toggle button, managed by the Page component if the Page isManagedSidebar prop is set to true */
   onNavToggle?: () => void;
+  /** Button id */
+  id?: string;
 }
 
 export const PageToggleButton: React.FunctionComponent<PageToggleButtonProps> = ({
   children,
   isNavOpen = true,
   onNavToggle = () => undefined as any,
+  id = 'nav-toggle',
```

Comment:
It shouldn't conflict, because the PageToggleButton is intended for use in the Masthead, and in those cases you are not using the PageHeader component.